app-main/public/js/init/showContainers.test.js

```diff
@@ -0,0 +1,27 @@
+import showContainers from "./showContainers.js";
+
+test("Tests adjusting/removing classes from container elements", () => {
+  document.body.innerHTML = `
+    <div class="pre-auth-container"></div>
+    <div class="profile"></div>
+    <div class="app-container"></div>
+    <div class="modal"></div>
+    <div class="overlay"></div>
+  `;
+
+  const preAuthContainer = document.querySelector(".pre-auth-container");
+  const profile = document.querySelector(".profile");
+  const appContainer = document.querySelector(".app-container");
+  const modal = document.querySelector(".modal");
+  const overlay = document.querySelector(".overlay");
+
+
+  showContainers();
+
+  expect(preAuthContainer.attributes.hidden.value).toBe("");
+  expect(profile.attributes.hidden).toBe(undefined);
```

Comment:
Unless I'm mistaken, it looks like we need to add the `hidden` attribute to the test markup (using `index.html` as a reference). Otherwise this doesn't really test anything. If I place `expect(profile.attributes.hidden.value).toBe("");` before the `showContainers()` call I'd expect this to be true but it fails.

Reply:
Great catch, it absolutely should be in the test markup.  I've added the hidden attribute to both the "profile" div's markup as well as the "app-container" div's markup.  If those two assertions are moved before the showContainers() call, the tests fail as the attribute is defined.  I believe this should fix the issue.